tests/Browser/pages/Page.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Tests\Browser\Pages;
+
+use Laravel\Dusk\Page as BasePage;
+
+abstract class Page extends BasePage
+{
+    /**
+     * Get the global element shortcuts for the site.
+     *
+     * @return array
+     */
+    public static function siteElements()
+    {
+        return [
+            '@element' => '#selector',
```

Comment:
I was reading about this in the docs and other posts and still a little confused how it knows what the `selector` is - would you mind explaining this?

Reply:
These are basically just a "table of contents" for CSS selectors on a page.

So for example, I could put a selector like `.header__title` (or something really hairy like `#campaign-inbox .table-row:first-of-type .table-cell:first-of-type`) and reference it by a friendlier name like `@title` or `@first-result` in my tests.

Reply:
ohhh we have an `id` that is `selector`? I see! For some reason I thought that was a placeholder word or something so was confused how it knew what the `selector` is on the page. Thanks, @DFurnes !

Reply:
Oh sorry, no! This was an example included as part of the default Dusk scaffolding. I'll remove it so there isn't any confusion. 💥

Reply:
ohhhh got it! Thanks @DFurnes !